pkg/netrc/testdata/key.json

```diff
@@ -1,3 +1 @@
-{
-    "test-key": "test-value"
-}
\ No newline at end of file
+{"test-key": "test-value"}
```

Comment:
Any chance we can revert this change? This is unrelated to the fix and the original key.json wasn't incorrectly formatted either.

Reply:
Without this change, tests in Windows are not passing due to new line differences. It was a quick fix to mitigate that. Should I revert and leave the tests broken in Windows or leave like this?

Reply:
Can you explain more? I thought this file was only used in the `add the first location with json key` test case?

Reply:
Yes! It took me quite some time to understand why tests were passing on Mac/Linux and not in Windows, then I figured out that was the reason. And I just got rid of newlines since they are not too related to what we are trying to test.